.github/workflows/scripts/is_latest_release.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import argparse
+
+from version_utils import getVersionTags, tryParseVersion
```

Comment:
What is a 'latest release'?

When we release a build, isn't that by definition the latest released since we only move versions forward?